### PR TITLE
Desul Export Target

### PR DESCRIPTION
**Pull request by mdavis36**
# Summary

- This PR is a bugfix
  - It exports desul cmake targets when RAJA_ENABLE_DESUL_ATOMICS is enabled

### CODE REVIEW

**Comment by artv3** [comment]
@mdavis36 should this be on the list: https://github.com/LLNL/RAJA/issues/1387 ?

**Review by rhornung67 — APPROVED** [review]
@mdavis36 looks OK to me. But, others with more CMake knowledge than me should approve before merging.

**Comment by rhornung67** [comment]
@davidbeckingsale please review this PR when you have a minute. It looks OK to me, but someone with more CMake knowledge than me should approve before merging.

**Comment by davidbeckingsale** [comment]
Looks good to me. @mdavis36 - if you remove the part of this conditional checking for "RAJA_ENABLE_DESUL_ATOMICS" then the export will be tested as part of our CI

https://github.com/LLNL/RAJA/blob/efb08ec8e4bbb9346183658ce651e86d17b5ebb5/scripts/gitlab/build_and_test.sh#L228

File: share/raja/cmake/RAJA-config.cmake.in
Comment (rhornung67) [inline share/raja/cmake/RAJA-config.cmake.in:35]:
```suggestion
# cache the prefix dir (could be overriden by find_dependency)
```

**Comment by rhornung67** [comment]
@davidbeckingsale did you want anyone else to look at this, or should we merge it?

**Comment by rhornung67** [comment]
Sorry. Mis-click.

**Comment by davidbeckingsale** [comment]
> @davidbeckingsale did you want anyone else to look at this, or should we merge it?

Wouldn't mind if @trws had a chance...

**Comment by rhornung67** [comment]
Thanks @trws

**Pull request merged**
